:whale: Add dockerfile for setting up deploy in kintohub :tada: (#8)
piterpy-meetup/c4p2n: README.md

_Created during the first B360R070N session._

**c4p2n** (Call For Paper To Notion) is a webhook that submits data about the talk from TypeForm to our Notion.

## Deployment

> We currently use awesome [kintohub](https://kintohub.com) as hosting for our docker container with web-app 

Reviewer: ```suggestion
We currently use awesome [kintohub](https://kintohub.com) to run a docker container with our web app. 
```
